src/ElementUtilities.js

```diff
@@ -21,7 +21,7 @@ const findClosest = (el, selector) => {
  * @return {boolean}        Whether table ancestor of 'el' is found
```

Comment:
On second thought, just go with renaming and adding that test and updating the assignment / parentheses usage.